Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Docker file is intended for the CI
 # A prerequisite is a published application in the .build/release  
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:2.2
+FROM mcr.microsoft.com/dotnet/core-nightly/aspnet:3.0
```

Comment:
I don't like nightlies, but I'd be fine with the standard 3.0 one that gets updated only on new previews at least.